test/common/testData.js

```diff
@@ -10,7 +10,7 @@ const SRMChallengeId = 1111111
 const SRMSubmissionId = 'bbbbbbbb-725d-4973-9778-360918a09bc0'
 const submitterHandle = 'lazybaer'
 const submitterEmail = 'email@domain.com.z'
-const reviewTypeId = '96059e8d-4761-4978-9a14-c86ae6b971c3'
+const reviewTypeId = '68c5a381-c8ab-48af-92a7-7a869a4ee6c3'
```

Comment:
should we just call to the `/v5/reviewTypes/` endpoint to pick this Id up by name?

Reply:
That's in the test data... The API calls are mocked there. We don't call the actual API.